packages/radix-ui-themes/src/components/card.tsx

```diff
@@ -26,7 +26,7 @@ const Card = React.forwardRef<CardElement, CardProps>((props, forwardedRef) => {
   function getChild() {
     const firstChild = React.Children.only(children) as React.ReactElement;
     return React.cloneElement(firstChild, {
-      children: firstChild.props.children,
+      children: <div className="rt-CardInner">{firstChild.props.children}</div>,
```

Comment:
instead of adding in 2 places, wouldn't it make more sense to wrap around line 46 like below?